samples/ControlCatalog/Pages/DataGridPage.xaml

```diff
@@ -6,6 +6,43 @@
     <local:GDPValueConverter x:Key="GDPConverter" />
   </UserControl.Resources>
   <UserControl.Styles>
+    <!-- The draft fix works only when DataGridFrozenGrid.IsFrozen is set to false, so the style changes that settings in order to present
```

Comment:
Obviously a case with `DataGridFrozenGrid.IsFrozen` set to `true` should be covered as well.